src/js/modules/SelectRange/SelectRange.js

```diff
@@ -908,27 +910,29 @@ export default class SelectRange extends Module {
 		return range;
 	}
 	
-	resetRanges() {
+	resetRanges(forceNewRange = true) {
```

Comment:
I'm not sure If I like adding a `forceNewRange` flag like this. This could be a sign that we probably need to make a new function. Is it possible to remove this?